src/main/resources/files/gradle/liberty/build.gradle.tpl

```diff
@@ -1,6 +1,8 @@
 apply plugin: 'war'
 apply plugin: 'liberty'
 
+def projectName = 'demo'
```

Comment:
This should not be hard coded. It should be whatever the project end users put in.

Reply:
It is now set to the artifact id.